notesapi/v1/search_indexes/backends/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .note import CompoundSearchFilterBackend
+
+__all__ = ('CompoundSearchFilterBackend',)
```

Comment:
Open edX style is usually to not define `__all__`, but doing it is fine. Just calling out that you don't need to.

Reply:
Thanks. I got it. I thought it was a good solution to explicitly define the interface of this module.